From pavel@ucw.cz  Fri Jul 17 10:57:36 2009
From: Pavel Machek <pavel@ucw.cz>
Date: Fri, 17 Jul 2009 14:55:53 +0200
Subject: Staging: HTC Dream: Makefile glue
To: greg@kroah.com, swetland@google.com
Message-ID: <20090717125553.GA1920@elf.ucw.cz>
Content-Disposition: inline

From: Pavel Machek <pavel@ucw.cz>

This provides Makefile/Kconfig glue for HTC Dream staging parts.

Signed-off-by: Pavel Machek <pavel@ucw.cz>
Cc: Brian Swetland <swetland@google.com>
Cc: Iliyan Malchev <ibm@android.com>
Cc: San Mehat <san@android.com>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>


---
 drivers/staging/Kconfig  |    4 ++++
 drivers/staging/Makefile |    2 ++
 2 files changed, 6 insertions(+)

--- a/drivers/staging/Kconfig
+++ b/drivers/staging/Kconfig
@@ -133,6 +133,10 @@ source "drivers/staging/cpc-usb/Kconfig"
 
 source "drivers/staging/pata_rdc/Kconfig"
 
+source "drivers/staging/dream/smd/Kconfig"
+
+source "drivers/staging/dream/camera/Kconfig"
+
 source "drivers/staging/udlfb/Kconfig"
 
 endif # !STAGING_EXCLUDE_BUILD
--- a/drivers/staging/Makefile
+++ b/drivers/staging/Makefile
@@ -49,3 +49,5 @@ obj-$(CONFIG_VT6656)		+= vt6656/
 obj-$(CONFIG_USB_CPC)		+= cpc-usb/
 obj-$(CONFIG_RDC_17F3101X)	+= pata_rdc/
 obj-$(CONFIG_FB_UDL)		+= udlfb/
+obj-$(CONFIG_MSM_ADSP)		+= dream/qdsp5/ dream/smd/
+obj-$(CONFIG_MSM_CAMERA)	+= dream/camera/
